presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -805,6 +805,27 @@ Argument Name         required   type            Description
 ``snapshot_id``       ✔️          long            Snapshot ID to rollback to
 ===================== ========== =============== =======================================================================
 
+Rollback to Timestamp
+^^^^^^^^^^^^^^^^^^^^
```

Comment:
```suggestion
^^^^^^^^^^^^^^^^^^^^^
```
Formatting line must match the number of characters in the line it is formatting.